nautobot_chatops/__init__.py

```diff
@@ -45,6 +45,10 @@ class NautobotChatOpsConfig(PluginConfig):
         # Mattermost-specific settings
         "mattermost_api_token": None,
         "mattermost_url": None,
+        # As requested on https://github.com/nautobot/nautobot-plugin-chatops/issues/114 this setting is used for
+        # sending all messages as an emphemeral message, meaning only the person interacting with the bot will see the
+        # responses.
+        "send_all_messages_emphemeral": False,
```

Comment:
`emphemeral` -> `ephemeral` throughout -- or maybe change to `private` instead? Not sure if "ephemeral" is widely used across platforms or if it's a Slack-ism?

Reply:
Updated to private.